src/mono/wasm/test-main.js

```diff
@@ -437,8 +468,8 @@ Promise.all([argsPromise, loadDotnetPromise]).then(async ([_, createDotnetRuntim
 });
 
 const App = {
-    init: async function ({ MONO, INTERNAL, BINDING, Module, runArgs }) {
-        Object.assign(App, { MONO, INTERNAL, BINDING, Module, runArgs });
+    init: async function ({ MONO, INTERNAL, BINDING, IMPORTS, EXPORTS, Module, runArgs }) {
+        Object.assign(App, { MONO, INTERNAL, BINDING, IMPORTS, EXPORTS, Module, runArgs });
```

Comment:
this reinforces my distaste for choosing to manually enumerate all the properties of the dict this way so we have to touch a bunch of code in order to add a new one or rename one vs just passing the blob through as a single named argument and then unpacking it

Reply:
I plan to do JS API redesign soon. Let's talk about it then.